src/enums/entity-type.ts

```diff
@@ -4,6 +4,7 @@ import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
 export enum EntityType {
   // BC Entity Types:
   BC = CorpTypeCd.BC_COMPANY,
+  BEN = CorpTypeCd.BENEFIT_COMPANY,
```

Comment:
This may not be needed, depending on whether conversions are done in conversion-types.ts.

Reply:
Never mind. It's now needed.

Reply:
Is it better to keep it since it's referred by `origin_entity_type_cd`?

Reply:
Yes, or use the common mixin method to convert BC -> BEN (or however that works).

Karim, do you have a strong opinion on this? It's the old NR vs Entity type code mismatch thing.

Reply:
No strong opinions, both approaches are OK. I think keeping it as is should be fine Sev.